src/EFCore.PG/Migrations/NpgsqlMigrationBuilderExtensions.cs

```diff
@@ -11,6 +11,16 @@ namespace Microsoft.EntityFrameworkCore
     // TODO: This is unused. Can we remove it?
     public static class NpgsqlMigrationBuilderExtensions
     {
+        /// <summary>
+        /// Returns true if the active provider in a migration is the Npgsql provider.
+        /// </summary>
+        /// <param name="builder">The <see cref="MigrationBuilder" />.</param>
```

Comment:
Let's improve this since it doesn't provide any valuable information about the parameter. The description could be copy-pasted from one of EF Core providers (see [this](https://github.com/aspnet/EntityFrameworkCore/blob/24b9aa1d2e14fe2e737255ede9b2a7a623fcf2af/src/EFCore.Sqlite.Core/Extensions/SqliteMigrationBuilderExtensions.cs#L22-L25) for instance).

```suggestion
        /// The migrationBuilder from the parameters on <see cref="Migration.Up(MigrationBuilder)" /> or
        /// <see cref="Migration.Down(MigrationBuilder)" />.
```